<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student_inquiry</class>
 <widget class="QDialog" name="student_inquiry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>学生学业情况查询</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>661</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: qlineargradient(x1: 0, y1: 0, x2: 0.93, y2: 0.93,stop: 0 #FFBBFF, stop: 1 white);</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>660</y>
     <width>771</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2,10,3,2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>总学分：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_credit">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>null</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GPA:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_average_credit">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>null</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
